modules/setting/setting.go

```diff
@@ -718,8 +752,17 @@ func NewContext() {
 			delete(SSH.MinimumKeySizes, strings.ToLower(key.Name()))
 		}
 	}
+
 	SSH.AuthorizedKeysBackup = sec.Key("SSH_AUTHORIZED_KEYS_BACKUP").MustBool(true)
 	SSH.CreateAuthorizedKeysFile = sec.Key("SSH_CREATE_AUTHORIZED_KEYS_FILE").MustBool(true)
+
+	SSH.AuthorizedPrincipalsBackup = false
```

Comment:
I just updated this if statement, as previously it was `if AuthorizedPrincipalsEnabled then hardcode false` whereas now it will accept settings if principals are enabled.